### ISSUE TITLE
Config file not generated correctly

### ISSUE THREAD

**Issue by kenbaird13** [ChiselsAndBits/Chisels-and-Bits#1082]
Whenever I boot up my server, I get this warning: com.google.gson.JsonParseException: Vector4f: settings.placement.color.missing-bits-or-space is not an array, or an object with x,y,z,w as parameters. And when I change the field names from "field_XXXXX" to X,Y,Z or W, the error goes away. This is a problem because when someone joins it tries to sync config, and crashes

**Comment by Aterniad** [comment]
Exactly same issue here. Tried different versions and no luck
```
[main/ERROR]: General failure during configuration parsing
com.google.gson.JsonParseException: Vector4f: settings.placement.color.success is not an array, or an object with x,y,z,w as parameters.
	at com.communi.suggestu.scena.fabric.platform.configuration.FabricConfigurationBuilder.lambda$defineVector4f$1(FabricConfigurationBuilder.java:93) ~[scena-1.0.84-ed30b19c73a0ca5a.jar:?]
	at com.communi.suggestu.scena.fabric.platform.configuration.FabricConfigurationValue.get(FabricConfigurationValue.java:44) ~[scena-1.0.84-ed30b19c73a0ca5a.jar:?]
	at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?]
	at com.communi.suggestu.scena.fabric.platform.configuration.FabricConfigurationSpec.forceGetAll(FabricConfigurationSpec.java:36) ~[scena-1.0.84-ed30b19c73a0ca5a.jar:?]
	at com.communi.suggestu.scena.fabric.platform.configuration.FabricConfigurationManager.lambda$createBuilder$5(FabricConfigurationManager.java:91) ~[scena-1.0.84-ed30b19c73a0ca5a.jar:?]
	at com.communi.suggestu.scena.fabric.platform.configuration.FabricConfigurationBuilder.setup(FabricConfigurationBuilder.java:245) ~[scena-1.0.84-ed30b19c73a0ca5a.jar:?]
	at mod.chiselsandbits.config.ClientConfiguration.(ClientConfiguration.java:79) ~[chisels-and-bits-fabric-1.3.122.jar:?]
	at mod.chiselsandbits.config.ChiselsAndBitsConfiguration.(ChiselsAndBitsConfiguration.java:11) ~[chisels-and-bits-fabric-1.3.122.jar:?]
	at mod.chiselsandbits.ChiselsAndBits.(ChiselsAndBits.java:34) ~[chisels-and-bits-fabric-1.3.122.jar:?]
	at mod.chiselsandbits.fabric.Fabric.lambda$new$2(Fabric.java:25) ~[chisels-and-bits-fabric-1.3.122.jar:?]
	at com.communi.suggestu.scena.core.init.PlatformInitializationHandler.init(PlatformInitializationHandler.java:38) ~[scena-1.0.84-ed30b19c73a0ca5a.jar:?]
	at com.communi.suggestu.scena.fabric.Fabric.onInitialize(Fabric.java:20) ~[scena-1.0.84-ed30b19c73a0ca5a.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) [fabric-loader-0.14.14.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35) [fabric-loader-0.14.14.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:62) [fabric-loader-0.14.14.jar:?]
	at net.minecraft.server.Main.main(Main.java:101) [server-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) [fabric-loader-0.14.14.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.14.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.14.14.jar:?]
	at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69) [fabric-loader-0.14.14.jar:?]
```
Version 1.19.2

**Comment by marchermans** [comment]
Fixed in 1.3.334
